CMakeLists.txt

```diff
@@ -165,6 +165,22 @@ if(NOT APPIMAGE_BUILD)
     add_subdirectory(tests)
 endif()
 
+install(
+    FILES com.kdab.Hotspot.xml
```

Comment:
is the uppercase `Hotspot` needed, or could this remain lowercase?

Reply:
I tried to match the naming of the desktop file, but I looked at the appdata file.